types/mjml-react/index.d.ts

```diff
@@ -125,43 +130,62 @@ export class MjmlStyle extends React.Component<{ children?: string | undefined,
 export class MjmlTitle extends React.Component<{ children?: string | undefined }> { }
 
 // mj-accordion
-export class MjmlAccordion extends React.Component<MjmlAccordionElementProps> { }
+export interface MjmlAccordionProps {
+    children?: React.ReactNode;
+    border?: string | undefined;
+    containerBackgroundColor?: React.CSSProperties['backgroundColor'] | undefined;
+    fontFamily?: string | undefined;
+    iconAlign?: string | undefined;
+    iconHeight?: string | undefined;
+    iconPosition?: 'left' | 'right' | undefined;
+    iconUnwrappedAlt?: string | undefined;
+    iconUnwrappedUrl?: string | undefined;
+    iconWidth?: string | undefined;
+    iconWrappedAlt?: string | undefined;
+    iconWrappedUrl?: string | undefined;
+}
+
+export class MjmlAccordion extends React.Component<MjmlAccordionProps & PaddingProps & ClassNameProps> { }
```

Comment:
The props for `MjmlAccordion` differ slightly from `MjmlAccordionElement` despite the one definition being shared by the 2 components here originally. I split them out (hence the chonkiness, and perhaps why it looks like there's autoformatting). Saying that, I can shift the common props into a separate type and union that.

Case in point [Accordion](https://documentation.mjml.io/#mj-accordion) has padding props, `containerBackgroundColor`, etc however [AccordionElement](https://documentation.mjml.io/#mj-accordion-element) does not


https://documentation.mjml.io/#mj-accordion-element

Reply:
I think it makes sense to have separate props for these components since they are not guaranteed to change together. I'm concerned with backwards compatibility, specifically removal or props from `MjmlAccordionElementProps` and props that are not in `MjmlAccordionProps` that used to be in `MjmlAccordionElementProps`. I think cleaning up the diff in the `MjmlAccordionElementProps` props will make this a lot easier to tell.